### PR TITLE
[TH] Copyright Translation

### PR DESCRIPTION
**Pull request by steamhrk**
Adding Thai translation on copyright page.

### CODE REVIEW

File: wiki/Legal/Copyright/th.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+4. ชื่อ ที่อยู่ และหมายเลขโทรศัพท์ของผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งาน รวมถึงข้อความที่ระบุว่าผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งานยินยอมอยู่ภายใต้อำนาจศาลแห่งรัฐบาลกลางของสหรัฐอเมริกาสำหรับเขตอำนาจศาลในการพิจารณาคดีในเขตที่ผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งานพำนักตามที่อยู่ทางกายภาพ หรือหากว่าที่อยู่ของผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งานไม่ได้อยู่ในประเทศสหรัฐอเมริกา ก็ยินยอมอยู่ใต้อำนาจศาลสำหรับการพิจารณาคดีในเขตใดก็ตามที่ osu! ดำเนินการอยู่ และผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งานจะยอมรับกระบวนการส่งหมายศาลจากบุคคลที่ดำเนินการยื่นคำบอกกล่าวการละเมิดลิขสิทธิ์ของเนื้อหาข้อมูลที่กล่าวอ้างหรือตัวแทนของบุคคลนั้น ๆ
+
+เมื่อหนังสือโต้แย้งได้ส่งถึงตัวแทนที่ได้รับมอบหมาย osu! อาจส่งสำเนาหนังสือโต้แย้งไปยังฝ่ายร้องเรียน เพื่อแจ้งให้ทราบว่าบุคคลนั้นสามารถแทนที่เนื้อหาที่ถูกลบหรือปิดกั้นการใช้งานได้ภายใน 10 วันทำการ เว้นแต่เจ้าของลิขสิทธิ์จะใช้สิทธิดำเนินการทางศาลต่อผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งาน เนื้อหาที่ถูกลบอาจถูกแทนที่หรือถูกกู้คืนมาภายใน 10 ถึง 14 วันทำการหรือมากกว่านั้นหลังจากวันที่ได้รับหนังสือโต้แย้ง ทั้งนี้เป็นไปตามดุลยพินิจของ osu!
```
Comment (ZephyrousZ) [inline wiki/Legal/Copyright/th.md]:
```suggestion
เมื่อหนังสือโต้แย้งได้ส่งถึงตัวแทนที่ได้รับมอบหมาย osu! อาจส่งสำเนาหนังสือโต้แย้งไปยังฝ่ายรับเรื่องร้องเรียน เพื่อแจ้งให้ทราบว่าบุคคลผู้นั้นสามารถแทนที่เนื้อหาที่ถูกลบหรือปิดกั้นการใช้งานได้ภายใน 10 วันทำการ เว้นแต่เจ้าของลิขสิทธิ์จะใช้สิทธิดำเนินการทางศาลต่อผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งาน เนื้อหาที่ถูกลบอาจถูกแทนที่หรือถูกกู้คืนกลับมาภายใน 10 ถึง 14 วันทำการหรืออาจมากกว่านั้นหลังจากวันที่ได้รับหนังสือโต้แย้ง ทั้งนี้ให้เป็นไปตามดุลยพินิจของ osu!
```

**Comment by ZephyrousZ** [comment]
Overall is pretty good, understandable. :D

File: wiki/Legal/Copyright/th.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+**ที่อยู่**:
+
+```
+Dean Herbert
+41 Gregory Street
+Wembley, WA, 6014
+Australia
+```
+
+**ที่อยู่ (ภาษาไทย)**:
```
Comment (cl8n) [inline wiki/Legal/Copyright/th.md]: this seems unnecessary to include

File: wiki/Legal/Copyright/th.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+**การดำเนินการของ osu!:**
+
+1. เพื่อทำการลบหรือปิดกั้นการเข้าถึงเนื้อหาที่ระบุในหนังสือร้องเรียนการละเมิดลิขสิทธิ์
+
+2. เพื่อทำการแจ้งผู้ให้บริการเนื้อหา สมาชิก หรือผู้ใช้งานว่าได้ลบหรือปิดกั้นการเข้าถึงเนื้อหานั้น และ
+
+3. เพื่อทำการระงับหรือยุติบัญชีผู้ใช้ตามความเหมาะสม หากผู้ใช้งานทำการละเมิดซ้ำอีก
+
+## การส่งหนังสือโต้แย้งการละเมิดลิขสิทธิ์ (Counter-Notice)
```
Comment (Joehuu) [inline wiki/Legal/Copyright/th.md]: Do we need the `(Counter-Notice)`?

Comment (TPGPL) [reply wiki/Legal/Copyright/th.md]: prob

**Pull request merged**